### DISCUSSION TITLE
How to integrate Carbon Fields with Rank Math?

### DISCUSSION THREAD

**Discussion by vladbv** [htmlburger/carbon-fields#1136] [Q&A]
Hello, I am trying to integrate Carbon Fields with Rank Math properly. How exactly do you integrate Carbon Fields with Rank Math the proper way? If I turn on the "Custom Fields" module under the "Advanced Settings" tab in Rank Math, will that be enough to index the custom Carbon Fields? I've used and have been looking at [https://github.com/htmlburger/carbon-fields-yoast](https://github.com/htmlburger/carbon-fields-yoast), which is great for Yoast integration, but do you follow the same pattern for Rank Math?

Please, help, thank you!

**Comment by sabrysuleiman** [comment]
I’ve made a simple snippet that could assist
https://gist.github.com/sabrysuleiman/b730a0efa7fb4dfc55d1d4f0532b7534
